resteasy-jaxrs/src/main/java/org/jboss/resteasy/plugins/providers/sse/SseEventInputImpl.java

```diff
@@ -118,11 +120,11 @@ public InboundSseEvent read() throws IllegalStateException
             {
                temSave.write(b);
                b = readLine(entityStream, ':', temSave);
-               String fieldName = temSave.toString(charset.toString());
+               String fieldName = temSave.toString(StandardCharsets.UTF_8.name());
```

Comment:
@jimma, did you mean to change charset.toString() to charset.name() here? You don't want to hard code UTF-8, do you?

Reply:
Oh, wait, the spec says
```
Event streams in this format must always be encoded as UTF-8.
```
Should we just ignore charsets?

Reply:
What do you mean by ignore charsets ?  temSave.toString() will use platform default character set.